Make check-pypi-packaging -- permission denied

$ make check-pypi-packaging 
```
podman run --rm -ti -v /home/flachman/Projects/TomasTomecek/ansible-bender:/src -w /src knife bash -c '\
	set -x \
	&& rm -f dist/* \
	&& python3 ./setup.py sdist bdist_wheel \
	&& pip3 install dist/*.tar.gz \
	&& ansible-bender --help \
	&& ansible-bender build --help \
	&& pip3 show ansible-bender \
	&& twine check ./dist/* \
	&& python3 -c "import ansible_bender; ansible_bender.__version__.startswith(\"0.3.1\")" \
	&& pip3 show -f ansible-bender | ( grep test && exit 1 || :) \
	'
+ rm -f 'dist/*'
+ python3 ./setup.py sdist bdist_wheel
(null): can't open file './setup.py': [Errno 13] Permission denied
make: *** [Makefile:15: check-pypi-packaging] Chyba 2
```

The mounted probably volumes needs `:z`/`:Z` to get the permissions...

(If this is the right solution, I'll send a PR.)

--- a/Makefile
+++ b/Makefile
@@ -9,10 +9,10 @@ check:
 	PYTHONPATH=$(CURDIR) pytest-3 -v $(TEST_TARGET)
 
 shell-in-knife:
-	podman run --rm -ti -v $(CURDIR):/src -w /src $(KNIFE) bash
+	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(KNIFE) bash
 
 check-pypi-packaging:
-	podman run --rm -ti -v $(CURDIR):/src -w /src $(KNIFE) bash -c '\
+	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(KNIFE) bash -c '\
 		set -x \
 		&& rm -f dist/* \
 		&& python3 ./setup.py sdist bdist_wheel \
@@ -30,7 +30,7 @@ check-pypi-packaging:
 #       run tests as an unpriv user
 # TODO: podman inside needs to use vfs storage driver
 check-smoke:
-	sudo podman run --net=host --rm -ti -v $(CURDIR):/src -w /src registry.fedoraproject.org/fedora:29 bash -c '\
+	sudo podman run --net=host --rm -ti -v $(CURDIR):/src:Z -w /src registry.fedoraproject.org/fedora:29 bash -c '\
 		dnf install -y buildah podman \
 		&& podman pull docker.io/library/python:3-alpine \
 		&& pip3 install . \

